app.js

```diff
@@ -372,7 +374,10 @@ var ontorrent = function (torrent) {
       var peerslisted = 0
 
       clivas.clear()
-      if (argv.airplay) clivas.line('{green:streaming to} {bold:apple-tv} {green:using airplay}')
+      if (argv.airplay) {
+        if (airplayServer) clivas.line('{green:streaming to} {bold:' + airplayServer.name + '} {green:using airplay}')
+        else clivas.line('{green:streaming} {green:using airplay}')
+      }
       else clivas.line('{green:open} {bold:' + (player || 'vlc') + '} {green:and enter} {bold:' + href + '} {green:as the network address}')
```

Comment:
you have to put curly brackets around this else when you put curly around the if according to standard